vite.config.dev.ts

```diff
@@ -43,8 +46,10 @@ async function getConfig(): Promise<UserConfig> {
         hook: 'writeBundle'
       }),
     ],
+    envDir: "udata_front/theme/gouvfr/datagouv-components",
     build: {
       minify: false,
+      cssCodeSplit: false,
```

Comment:
We don't want css splitting because we are doing the import ourself without manifest